base/doc/ltnews41.tex

```diff
@@ -244,6 +244,14 @@ \subsection{Socket and plug conditionals}
 %
 \githubissue{1577}
 
+\subsection{Accessing the current counter}
+Counter commands such as \cs{alph}, \cs{stepcounter}, may now have
+argument \texttt{*} to denote the current counter (as used by
+\cs{label}).  This is compatible with the \pkg{enumitem} use of
```

Comment:
with the package \pkg{enumitem}

Reply:
I _keep_ doing that (reading `pkg` as package so not adding it)